cura/OAuth2/KeyringAttribute.py

```diff
@@ -4,6 +4,7 @@
 
 import keyring
 from keyring.backend import KeyringBackend
+from keyring.backends.macOS.api import KeychainDenied
```

Comment:
Should this be a platform-dependent import?

On Windows, I am getting the following error on this import `FileNotFoundError: Could not find module 'C:\System\Library\Frameworks\Security.framework\Versions\A\Security' (or one of its dependencies). Try using the full path with constructor syntax.`

Reply:
It should indeed. I changed it in https://github.com/Ultimaker/Cura/pull/10010/commits/951c656096e4bc94d3fdc892b432d6b65c2715f0.